relayer29/relayer.go

```diff
@@ -35,8 +35,12 @@ func (r *Relay) Storage(ctx context.Context) eventstore.Store {
 	}
 }
 
-func (r *Relay) AcceptEvent(ctx context.Context, ev *nostr.Event) bool {
-	return true
+func (r *Relay) AcceptEvent(ctx context.Context, ev *nostr.Event) (bool, string) {
+	_, ok := relayer.GetAuthStatus(ctx)
+	if !ok {
+		return false, "auth-required: please login"
+	}
+	return true, ""
```

Comment:
this sounds wrong, why would you always require login?

Reply:
I don't make sure but I thought this require always login since this provide private chat. Will revert this.